doc/readme.md

```diff
@@ -343,6 +343,7 @@ approval_prompt to `force`.
 | [options.state] | <code>String</code> |  | OAuth state parameter passed to the redirect uri. This parameter may be used for identifying the user who initiated the request. |
 | [options.forcePrompt] | <code>Boolean</code> | <code>false</code> | Setting `forcePrompt` to `true` will show the permissions approval screen on every authentication attempt, even if the user has previously consented to the exact scope of permissions. |
 | [options.vehicleInfo.make] | <code>Object</code> |  | `vehicleInfo` is an object with an optional property `make`. An optional parameter that allows users to bypass the car brand selection screen. For a complete list of supported makes, please see our [API Reference](https://smartcar.com/docs/api#authorization) documentation. |
+| [options.singleSelect] | <code>Boolean</code> | <code>false</code> | only allows users to select a single vehicle. Please refer to the [Single Select Guide](https://smartcar.com/docs/guides/single-select/). |
```

Comment:
Same as below, do you want to rephrase the wording?

Reply:
I'll discuss wording with Jeremy when he's back. I intentionally changed it because I found the other wording less user-friendly. I'll make sure all the SDKs have the same wording.

Reply:
Fixed

Reply:
Fixed